tests/acceptance/features/webUIFavorites/favoritesFile.feature

```diff
@@ -68,8 +68,8 @@ Feature: Mark file as favorite
     And the user opens folder "strängé नेपाली folder" using the webUI
     And the user marks file "lorem.txt" as favorite using the webUI
     When the user browses to the favorites page
-    Then file "lorem.txt" should be listed on the webUI
-    And file "simple-folder/lorem.txt" should be listed on the webUI
-    And folder "simple-empty-folder" should be listed on the webUI
-    And folder "simple-folder/simple-empty-folder" should be listed on the webUI
-    And file "strängé नेपाली folder/lorem.txt" should be listed on the webUI
+    Then file with path "lorem.txt" should be listed in the favorites page on the webUI
```

Comment:
since the user is already in the favourites page
```And file "lorem.txt" should be listed on the webUI``` should do the job

Reply:
"simple-folder/lorem.txt" is listed in favourites page, so what you did here is fine.